src/Tribe/Settings_Tab.php

```diff
@@ -270,6 +296,59 @@ public function do_content(): void {
 		}
 	}
 
+	/**
+	 * Adds a child tab to the current tab.
+	 *
+	 * @since TBD
+	 *
+	 * @param Tribe__Settings_Tab $tab The child tab to add.
+	 */
+	public function add_child( $tab ): void {
```

Comment:
This should have a typed parameter, then we don't need to filter the children later.

```suggestion
	public function add_child( Tribe__Settings_Tab $tab ): void {
```